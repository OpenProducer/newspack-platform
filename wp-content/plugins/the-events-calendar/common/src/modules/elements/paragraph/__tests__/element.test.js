import React from 'react';

import Paragraph, { SIZES } from '../element';

describe( '<Paragraph>', () => {
	test( 'default paragraph', () => {
		const component = renderer.create(
			<Paragraph>Modern Tribe</Paragraph>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );

	test( 'smaller paragraph', () => {
		const component = renderer.create(
			<Paragraph size={ SIZES.small }>Modern Tribe</Paragraph>,
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
