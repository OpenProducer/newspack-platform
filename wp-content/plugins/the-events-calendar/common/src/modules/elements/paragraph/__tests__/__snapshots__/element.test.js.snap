// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph> default paragraph 1`] = `
<p
  className="tribe-editor__paragraph tribe-editor__paragraph--medium"
>
  Modern Tribe
</p>
`;

exports[`<Paragraph> smaller paragraph 1`] = `
<p
  className="tribe-editor__paragraph tribe-editor__paragraph--small"
>
  Modern Tribe
</p>
`;
